frontend/source/js/components/timeline.js

```diff
@@ -148,6 +149,9 @@ export default Vue.component('timeline', {
       else if(s.startsWith('file.text')) {
         return 'text-entry';
       }
+      else if(s.startsWith('activity.watching')) {
+        return 'trakt-tile';
```

Comment:
I was going to build a similar tile for my home server (https://github.com/nicbou/homeserver). If this component is generic, I would just rename it to WatchEntry or something similar.

You don't need to do this. I'll make the changes myself if I ever add that functionality.